models/cow_protocol/ethereum/cow_protocol_ethereum_referrals.sql

```diff
@@ -0,0 +1,47 @@
+-- PoC Query: https://dune.com/queries/1789628?d=1
+WITH
+referral_map as (
+    select
+        distinct app_hash,
+        referrer
+    from {{ ref('cow_protocol_ethereum_app_data') }}
+    where referrer is not null
+)
+
+,trader_data as (
+    select
+        trader,
+        count(*) as num_trades,
+        sum(usd_value) as trader_volume
+    FROM {{ ref('cow_protocol_ethereum_trades') }}
+    group by trader
+)
```

Comment:
You don't use this anywhere.

Reply:
Right... thanks -- its used in one of the redundant queries I've been copy-pasting all across our [affiliate dashboard](https://dune.com/cowprotocol/affiliate-referrals) (check it out), probably got caught up in this... will remove.

Reply:
Fixed. Thanks again for pointing this out!